docs/deploying/maincloud.md

```diff
@@ -0,0 +1,30 @@
+# Deploy to Maincloud
+
+Maincloud is a managed cloud service that provides developers an easy way to deploy their SpacetimeDB apps to the cloud.
+
+## Deploy via CLI
+
+1. Install the SpacetimeDB CLI for your platform: [Install SpacetimeDB](/install)
+1. Create your module (see [Getting Started](/docs/getting-started))
+1. Publish to maincloud:
```

Comment:
```suggestion
1. Publish to Maincloud:
```

NIT: I've been styling it with a capital M in branding, but `maincloud` in monospace fonts.